schema/deploy/shipping/views.sql

```diff
@@ -2131,4 +2147,170 @@ grant select
+          end as hcov19_result,
+          hcov19_result_release_date,
+          case
+              when puma in ('5310901', '5310902') then 'yakima'
+              when puma in ('5311701', '5311702', '5311703',
+                            '5311704', '5311705', '5311706') then 'snohomish'
+              when puma in ('5311601', '5311602', '5311603', '5311604',
+                            '5311605', '5311606', '5311607', '5311608',
+                            '5311609', '5311610', '5311611', '5311612',
+                            '5311613', '5311614', '5311615', '5311616') then 'king'
+          end as county
+
+      from shipping.scan_encounters_v1
+      join hcov19_presence_absence using (sample)
+
+    )
+
+    select
+        hcov19_result_release_date,
+        count(*) filter (where hcov19_result in ('positive', 'inconclusive')) as total_hcov19_positives,
+        count(*) filter (where hcov19_result in ('positive', 'inconclusive') and county = 'king') as king_county_positives,
+        count(*) filter (where hcov19_result in ('positive', 'inconclusive') and county = 'snohomish') as snohomish_county_positives,
+        count(*) filter (where hcov19_result in ('positive', 'inconclusive') and county = 'yakima') as yakima_county_positives,
+        count(*) filter (where hcov19_result in ('positive', 'inconclusive') and county is null) as other_positives
+    from scan_hcov19_results
+    group by hcov19_result_release_date
+;
+
+comment on view shipping.scan_hcov19_positives_v1 is
+  'A view of counts of hcov19 positives from the SCAN project grouped by date results were released.';
+
+-- Even if it's just aggregate counts of hcov19 positives,
+-- we should probably restrict access to this view to only hcov19-visibility
+-- and power-bi-exporter.
+--  -Jover, 9 July 2020
+revoke all on shipping.scan_return_results_v1 from reporter;
+
+grant select
+    on shipping.scan_return_results_v1
+    to "hcov19-visibility";
+
+revoke all
+    on shipping.scan_hcov19_positives_v1
+  from "power-bi-exporter";
+
+grant select
+    on shipping.scan_hcov19_positives_v1
+    to "power-bi-exporter";
+
+
+create or replace view shipping.seattle_neighborhood_districts_v1 as
+
+    select
+        identifier,
+        st_assvg(point) as centroid,
+        st_assvg(polygon) as svg_path
+    from warehouse.location
+    where scale = 'neighborhood_district'
+    and hierarchy -> 'city' = 'seattle'
+;
+
+comment on view shipping.seattle_neighborhood_districts_v1 is
+  'A view of Seattle neighborhood district polygons as SVG paths';
+
+revoke all
+    on shipping.seattle_neighborhood_districts_v1
+  from "power-bi-exporter";
+
+grant select
+    on shipping.seattle_neighborhood_districts_v1
+    to "power-bi-exporter";
+
+
+create or replace view shipping.scan_enrollments_v1 as
+
+    select
+        illness_questionnaire_date,
+        scan_study_arm,
+        priority_code,
+        puma,
+        centroid as seattle_neighborhood_district_centroid
```

Comment:
I suspect the centroid as an SVG path isn't useful to Formative here. I think just leaving the neighborhood identifier might be better.